gulp/tasks/icons.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (gulp, $, config) => {
+  const entryGlob = config.entryGlobs.icons;
+  const destPath = config.destPaths.icons;
+  const manifestDestPath = config.destPaths.revManifest;
+
+  const task = () => gulp.src(entryGlob)
+      .pipe($.svgSprite({
+        mode: {
+          symbol: {
+            dest: '',
```

Comment:
removed this 👍  The dest path changes to be `build/assets/icons/symbol/sprite.svg` but I think it is ok.